libglusterfs/src/inode.c

```diff
@@ -1708,11 +1709,18 @@ inode_table_with_invalidator(uint32_t lru_limit, xlator_t *xl,
         new->dentry_hashsize = dentry_hashsize;
     }
 
+    /* The size of hash table MUST always be power of 2 as required by hash_gfid
+     */
     if (inode_hashsize == 0) {
```

Comment:
Since very small hash tables don't really make much sense, probably we should enforce a minimum value now that we allow users to change it.

```suggestion
    if (inode_hashsize < 65536) {
```

Reply:
I'm very much OK with this, will update documentation too and deal with the formatting issues as suggested by ant-bot - how can I run those checks locally?

Reply:
You need to have clang-format installed. The you can run this:

<pre>
    diff <i>&lt;file&gt;</i> &lt;(clang-format --style=file <i>&lt;file&gt;</i>)
</pre>

Reply:
I usually run 'clang-format -i' on the code. More accurately:
```
git status |grep modified |awk '{print $NF}' |grep "\.c" |xargs clang-format -i
```